source/includes/steps-install-shell-base.yaml

```
@@ -52,7 +51,7 @@ content: |
      .. code-block:: sh
 
        sudo cp mongosh /usr/local/bin/
-       sudo cp mongocryptd-mongosh /usr/local/bin/
+       sudo cp mongosh_csfle_v1.so /usr/local/bin/
```

Might be more idiomatic to do this:

```suggestion
       sudo cp mongosh_csfle_v1.so /usr/local/lib/
```